frontend/src/components/Skeletons/SkeletonHomeSection.vue

```diff
@@ -13,33 +13,34 @@
   </v-col>
 </template>
 
-<script lang="ts">
-import { defineComponent } from 'vue';
+<script setup lang="ts">
+import { computed } from 'vue';
+import { useVuetify } from '@/composables';
```

Comment:
```suggestion
import { useDisplay } from 'vuetify';
```

If you read the comments of that composable, that composable is just meant to be used outside Vue components (i.e raw TS files). Here we could use Vuetify composables as stated in their docs.